postgres-express-setup-tutorial.md

```diff
@@ -19,70 +19,84 @@ summary = "The article is a checklist for myself, but for anyone else setting up
 
 {{% pin_it_image "postgres express tutorial" "img/posts/postgres-express-setup-tutorial/banner.jpg" "is-src-set" %}}
 
-The article is a checklist for myself, but for anyone else setting up PostgreSQL on MacOS for having a SQL database in a Express.js application. You can leave out the Express part to just set up Postgres on MacOS. If you attempt to run Postgres on Windows, I believe you can substitute most of the tools here for your machine.
-
-You may wonder: Why is there a need for another setup guide for Postgres in the first place? I found many tutorials and guides on how to set it up, but most of it was kinda blurry, outdated (old Postgres versions) and not to the point. So I wanted to have this little checklist for setting up Postgres 10 on MacOS for myself. If it is useful for you, it is an additional plus. The checklist is not comprehensive at all, maybe needs to be improved by using the correct terms (looking forward to your comments), but I will fill out the gaps along the way of using Postgres in Express myself.
+The article started as a checklist I put together for myself, which I've decided to share with everyone who needs help setting up PostgreSQL on MacOS, to use SQL database in a Express.js application. If you're just setting up Postgres on MacOS, you can leave out the part about Express. If you attempt to run Postgres on Windows, you should be able to substitute most of the tools here for your machine. You might be wondering why there's a need for another setup guide for Postgres in the first place, since there are a few across the web. I found many tutorials and guides on how to set it up, but found most of them to be unclear and outdated, based on old Postgres versions. The checklist is not comprehensive, though, as it pertains to my own way of doing things, so any feedback on simpler or more universal means is always welcome.
```

Comment:
"If you attempt to run Postgres on Windows, you should be able to substitute most of the tools here for your machine." -> "If you attempt to run Postgres on Windows, you will find guidance here as well."